Exercise_A/index.html

```diff
@@ -16,7 +16,7 @@
   </head>
   <body>
     <!-- CODE HERE -->
-
+    <a href="/Exercise_A/about.html"><img src="/assets/flowerShop.png" /> </a>
```

Comment:
My only comment on this exercise is that when you have an image within your document, you should add a property "alt" onto the image tag. 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img